models/User.js

```diff
@@ -0,0 +1,60 @@
+const { Model, DataTypes } = require('sequelize')
+const bcrypt = require('bcrypt')
+const sequelize = require('../config/connection')
+
+// create our User model
+class User extends Model {
+  // set up method to run on instance data (per user) to check password
+  checkPassword(loginPw) {
+    return bcrypt.compareSync(loginPw, this.password)
+  }
+}
+
+// create fields/columns for User model
+User.init(
+  {
+    id: {
+      type: DataTypes.INTEGER,
+      allowNull: false,
+      primaryKey: true,
+      autoIncrement: true
+    },
+    email: {
+      type: DataTypes.STRING,
+      allowNull: false
+    },
```

Comment:
Should probably validate it is an email address

```suggestion
    email: {
      type: DataTypes.STRING,
      allowNull: false,
      validate: {
        isEmail: true
      }
    },
```